### PR TITLE
Update PPP - Noel

### PR DESCRIPTION
**Pull request by noelmathewisaac**

### CODE REVIEW

File: docs/team/noelmathewisaac.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        * Justification: This feature makes it much easier for the user to categorise and filter out tasks belonging to a
+          certain category. By tagging tasks with relevant keywords, the user can later retrieve all the tasks from 
+          the same
+          cateogory really quickly instead of manually looking through all the tasks.
+        * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
+        * Relevant PRs: [\#36](https://github.com/AY2021S2-CS2103T-W14-4/tp/pull/36)
+    * #### UI Design and Enhancement
+      * What it does: Designed and enhanced the UI to make it more seamless and user friendly, while also being 
+        simple and functional.
+      * Justification: A great user interface is essential to the user expereince and will help in user retention
```
Comment (hengyongming) [inline docs/team/noelmathewisaac.md]: experience is spelled wrongly

File: docs/team/noelmathewisaac.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    * Managed the issue tracker and milestones v1.1 - v1.4 (4 milestones) on GitHub.
+
+* **Enhancements to existing features**:
+    * Reafactored AB3 code to be adapted for Taskify (Pull requests 
+      [\#31](https://github.com/AY2021S2-CS2103T-W14-4/tp/pull/31))
+    * Wrote additional tests for existing features to increase coverage.
+
+* **Documentation**:
+    * User Guide:
+        * Added glossary `delete` and `find`: [\#72]()
```
Comment (hengyongming) [inline docs/team/noelmathewisaac.md:50]: Link to github not included

Comment (noelmathewisaac) [reply docs/team/noelmathewisaac.md:50]: I am still updating haha. Can help merge this version first

Comment (hengyongming) [reply docs/team/noelmathewisaac.md:50]: Alright sure

**Review by hengyongming — APPROVED** [review]
Approving so that this PPP can be merge first

**Review by CSmortal — APPROVED** [review]
LGTM

**Pull request merged**